ckanext/switzerland/logic.py

```diff
@@ -177,34 +177,34 @@ def ogdch_clean_harvester_jobs(context, data_dict):
     tk.check_access('harvest_sources_clear', context, data_dict)
     model = context['model']
 
-    # get sources to clear from data_dict
+    # get sources from data_dict
     if 'harvest_source_id' in data_dict:
         harvest_source_id = data_dict['harvest_source_id']
         source = HarvestSource.get(harvest_source_id)
         if not source:
             log.error('Harvest source %s does not exist', harvest_source_id)
             raise NotFound('Harvest source %s does not exist' % harvest_source_id)
-        sources_to_clear = [source.id]
+        sources_to_cleanup = [source.id]
     else:
         all_sources = model.Session.query(HarvestSource.id).all()
-        sources_to_clear = list(itertools.chain(*all_sources))
+        sources_to_cleanup = list(itertools.chain(*all_sources))
 
-    log.info('Harvest sources clearing called for: {}'.format(','.join(sources_to_clear)))
+    log.info('Harvest job cleanup called for sources: {}'.format(','.join(sources_to_cleanup)))
 
     # get number of jobs to keep form data_dict
     if 'number_of_jobs_to_keep' in data_dict:
         number_of_jobs_to_keep = data_dict['number_of_jobs_to_keep']
     else:
-        log.error('Configuration missing for number of jobs to keep when clearing harvest sources')
-        raise ValidationError('Configuration missing for number of jobs to keep when clearing harvest sources')
+        log.error('Configuration missing for number of jobs to keep when cleaningup harvest jobs')
```

Comment:
tiny typo 'cleaningup'

Reply:
in general: so much better, thank you for the adaptations, now it is very 'clear' and 'clean' ;)